givitsite/coordinate/models.py

```diff
@@ -40,3 +41,19 @@ class Meta:
             'pickup_time',
             'drop_off_time'
             ]
+
+def create_new_coordinations(request):
```

Comment:
I'm not sure this code belongs in the model - it looks more like view code

Reply:
![image](https://user-images.githubusercontent.com/42575230/93184141-93ef6d00-f744-11ea-9799-318a79c478df.png)

I'm confused:)

maybe a bad function name?